src/Uno.UI.Tests/Windows_UI_Xaml_Data/xBindTests/Given_xBind_Functions.cs

```diff
@@ -9,61 +9,77 @@ public class Given_xBind_Functions
+				Assert.AreEqual(string.Empty, SUT._InstanceFunction_TwoParam.Text);
+				Assert.AreEqual(string.Empty, SUT._InstanceFunction_TwoParam_WithConstant.Text);
+				Assert.AreEqual(string.Empty, SUT._System_Function.Text);
+				Assert.AreEqual(string.Empty, SUT._System_Function_with_Quote.Text);
+				Assert.AreEqual(string.Empty, SUT._InstanceFunction_Parameterless.Text);
+				Assert.AreEqual(string.Empty, SUT._StaticFunction_Parameterless.Text);
+				Assert.AreEqual(string.Empty, SUT._InstanceFunction_Boolean_False.Text);
+				Assert.AreEqual(string.Empty, SUT._InstanceFunction_Boolean_True.Text);
+				Assert.AreEqual(string.Empty, SUT._StaticClass_PublicConstField.Text);
+				Assert.AreEqual(string.Empty, SUT._StaticPrivateConstField.Text);
+				Assert.AreEqual(string.Empty, SUT._InnerConstField.Text);
+				Assert.AreEqual("DefaultTextBoxText", SUT._ThrowingWithoutFallback.Text);
+				Assert.AreEqual("DefaultTextBoxText", SUT._ThrowingWithFallback.Text);
+				Assert.AreEqual("DefaultTextBoxText", SUT._NotThrowingWithoutFallback.Text);
+				Assert.AreEqual("DefaultTextBoxText", SUT._NotThrowingWithFallback.Text);
+
+				SUT.ForceLoaded();
+
+				Assert.AreEqual("42", SUT._InstanceProperty.Text);
+				Assert.AreEqual("43", SUT._StaticProperty.Text);
+				Assert.AreEqual("44", SUT._StaticPrivateProperty.Text);
+				Assert.AreEqual("45", SUT._StaticPrivateReadonlyField.Text);
+				Assert.AreEqual("-1", SUT._InstanceDP.Text);
+				Assert.AreEqual("Initial", SUT._InnerProperty.Text);
+				Assert.AreEqual("47", SUT._StaticClass_PublicStaticProperty.Text);
+				Assert.AreEqual("Hello world!", SUT.StaticPropertyDeepNestingOnNonStaticClass.Text);
+				Assert.AreEqual("52", SUT._InstanceFunction_OneParam.Text);
+				Assert.AreEqual("9", SUT._InstanceFunction_OneParam_DP_Update_OneWay.Text);
+				Assert.AreEqual("9", SUT._InstanceFunction_OneParam_DP_Update_OneTime.Text);
+				Assert.AreEqual("85", SUT._InstanceFunction_TwoParam.Text);
+				Assert.AreEqual("84.42", SUT._InstanceFunction_TwoParam_WithConstant.Text);
+				Assert.AreEqual("42, 43", SUT._System_Function.Text);
+				Assert.AreEqual("42, \'43\'", SUT._System_Function_with_Quote.Text);
+				Assert.AreEqual("Parameter-less result", SUT._InstanceFunction_Parameterless.Text);
+				Assert.AreEqual("Static Parameter-less result", SUT._StaticFunction_Parameterless.Text);
+				Assert.AreEqual("Was false", SUT._InstanceFunction_Boolean_False.Text);
+				Assert.AreEqual("Was true", SUT._InstanceFunction_Boolean_True.Text);
+				Assert.AreEqual("48", SUT._StaticClass_PublicConstField.Text);
+				Assert.AreEqual("46", SUT._StaticPrivateConstField.Text);
+				Assert.AreEqual("-5", SUT._InnerConstField.Text);
+				Assert.AreEqual("DefaultTextBoxText", SUT._ThrowingWithoutFallback.Text);
+				Assert.AreEqual("MyFallbackValue", SUT._ThrowingWithFallback.Text);
+				Assert.AreEqual("", SUT._NotThrowingWithoutFallback.Text);
+				Assert.AreEqual("", SUT._NotThrowingWithFallback.Text);
+
```

Comment:
@jeromelaban We should be getting the validation here with `				FeatureConfiguration.BindingExpression.HandleSetTargetValueExceptions = false;` at the beginning